{'turn_right':'move',
 'turn_left':'move',
 'go_straight':'move',
 'go_back':'move',
 'look_here':,'move',
 'start_learning':'learn',
 'start_follow':'follow',
 'stop_follow':'follow'}
